hikari/internal/time.py

```diff
@@ -52,7 +52,9 @@
 where `builtins.int` and `builtins.float` types are interpreted as a number of seconds.
 """
 
-DISCORD_EPOCH: typing.Final[int] = 1_420_070_400
+DISCORD_EPOCH: typing.Final[datetime.datetime] = datetime.datetime.fromtimestamp(
+    1_420_070_400, tz=datetime.timezone.utc
+)
 """Discord epoch used within snowflake identifiers.
 
 This is defined as the number of seconds between
```

Comment:
if you're going to make this a datetime rather than a timedelta then this docstring needs to be updated